### DISCUSSION TITLE
TransferList Group Selection Functionality

### DISCUSSION THREAD

**Discussion by afgprogrammer** [mantinedev/mantine#2696] [Q&A]
Hello, mantine community,

I wanna add a checkbox for group selection to the TransferList component, something like the below screenshot 

<img width="465" alt="Screen Shot 2022-10-12 at 11 42 30 PM" src="https://user-images.githubusercontent.com/17293422/195439112-5d4452fd-bd7a-4086-90f4-982708cd91e5.png">

if a group is selected all items of the group should be selected. I tried to add that with react `ref` but I couldn’t.

**Comment by afgprogrammer** [comment]
I am still looking for an answer

**Reply by rtivital** [reply]
It is not supported
